src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -0,0 +1,135 @@
+import datajoint as dj
+import numpy as np
+from ripple_detection import get_multiunit_population_firing_rate
+
+from spyglass.common import Session  # noqa: F401
+from spyglass.spikesorting.spikesorting_merge import SpikeSortingOutput
+from spyglass.spikesorting.unit_inclusion_merge import UnitInclusionOutput
+from spyglass.utils.dj_mixin import SpyglassMixin
+
+schema = dj.schema("sorted_spikes_group_v1")
+
+
+@schema
+class SortedSpikesGroup(SpyglassMixin, dj.Manual):
+    definition = """
+    -> Session
+    sorted_spikes_group_name: varchar(80)
+    """
+
+    class Units(SpyglassMixin, dj.Part):
+        definition = """
+        -> master
+        -> UnitInclusionOutput.proj(unit_inclusion_merge_id='merge_id')
+        """
+
+    def create_group(
+        self,
+        group_name: str,
+        nwb_file_name: str,
+        unit_inclusion_merge_ids: list[str],
+    ):
+        group_key = {
+            "sorted_spikes_group_name": group_name,
+            "nwb_file_name": nwb_file_name,
+        }
+        self.insert1(
+            group_key,
+            skip_duplicates=True,
+        )
+        for unit_inclusion_merge_id in unit_inclusion_merge_ids:
+            self.Units.insert1(
+                {
+                    "unit_inclusion_merge_id": unit_inclusion_merge_id,
+                    **group_key,
+                },
+                skip_duplicates=True,
+            )
```

Comment:
I try to run inserts in as few statements as possible to keep it in the same transaction that dj will roll back if errors

```suggestion
        self.Units.insert(
            [
                {
                    "unit_inclusion_merge_id": id,
                    **group_key,
                }
                for id in unit_inclusion_merge_ids
            ],
            skip_duplicates=True,
        )
```